pkgs/by-name/va/vapoursynth-knlmeanscl/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  pkg-config,
+  boost,
+  vapoursynth,
+  opencl-headers,
+  ocl-icd,
+}:
+
+stdenv.mkDerivation {
+  pname = "vapoursynth-knlmeanscl";
+  version = "unstable-2023-06-05";
+
+  src = fetchFromGitHub {
+    owner = "Khanattila";
+    repo = "KNLMeansCL";
+    rev = "ca424fa91d1e16ec011f7db9c3ba0d1e76ed7850";
+    hash = "sha256-co8Jaup3bvvJaKw830CqCkAKHRsT5rx/xAYMbGhrMRk=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+  ];
+
+  buildInputs = [
+    boost
+    vapoursynth
+    ocl-icd
+    opencl-headers
+  ];
+
+  postPatch = ''
+    rm -rf KNLMeansCL/vapoursynth
+    ln -s ${vapoursynth}/include/vapoursynth KNLMeansCL/vapoursynth
```

Comment:
Is it even necessary to link the include directory here?

Specifying `vapoursynth` in `buildInputs` might already pull in the correct include path through the compiler wrapper.